### PR TITLE
Add a utility function to create tabs with lazy loading functionality

### PR DESCRIPTION
**Pull request by ogmaresca**
Also update the What's New and Merge tabs to use the new function. This means that the app will not reach out to Github for the changelog until the user clicks on the What's New tab

### CODE REVIEW

File: ui/media/js/utils.js [hunk 1]
```diff
@@ -720,9 +720,140 @@ Array.prototype.addEventListener = function(method, callback) {
     const originalFunction = this[method]
     if (originalFunction) {
         this[method] = function() {
-            console.log(`Array.${method}()`, arguments)
```
Comment (ogmaresca) [inline ui/media/js/utils.js:723]: Removed the console.log fromhttps://github.com/cmdr2/stable-diffusion-ui/pull/1087

**Comment by cmdr2** [comment]
@ogmaresca  Apologies for the delay in reviewing this. Is this blocking any plugin-related work for you? Or is the main intent here to reduce the loading time?

Thanks

**Comment by ogmaresca** [comment]
@cmdr2 I do have a plugin I'm working on that would benefit from this method. Plus, I think that reducing the amount of DOM manipulations a plugin needs to make is useful to prevent updates from breaking plugins

**Comment by cmdr2** [comment]
@ogmaresca Sorry for the extreme delay. Merged, thanks for the PR!

**Comment by cmdr2** [comment]
@ogmaresca This is now in the main branch

**Pull request merged**